encryption_works.md

```diff
@@ -337,6 +337,22 @@ Washington Post journalist Barton Gellman, who Edward Snowden trusted with infor
 
 Snowden was right to be cautious and to insist that he check Gellman's new PGP fingerprint. PGP, if used right, provides the tools necessary to prevent MITM attacks. But these tools only work if the users are vigilant about identity verification.
 
+### What PGP Doesn't Do
```

Comment:
Really like this section, in general. We could link to someone in government claiming that metadata is as important or more important than content. Also Michael Hayden saying "we kill people based on metadata." Also could add another example in there from EFF's why metadata matters post. https://www.eff.org/deeplinks/2013/06/why-metadata-matters